scripts/run.sh

```diff
@@ -14,7 +14,7 @@ umask 000 # Files and folders we create should be world accessible
 while true 
 do
     echo "Starting scdaq..."
-    ./scdaq 2>&1 | logger --tag scdaq --id -p user.debug
+    /opt/scdaq/bin/scdaq /etc/scdaq/scdaq.conf 2>&1 | logger --tag scdaq --id -p user.debug
```

Comment:
Sure it can be improved but at least this one is consistent with the actual location of the configuration file
I would also like to use something else than a pipe to the logger command for that matter (and not write in the journal)